utils.go

```diff
@@ -557,3 +557,21 @@ func concat(slices ...[]string) []string {
 	}
 	return slice
 }
+
+func isReleaseConsideredToRun(r *release) bool {
```

Comment:
nit: Maybe we could make this a method of the release object instead so we could do `r.IsConsideredToRun()`

Reply:
I should have thought about that, applied the change, thanks. :-)